mitxpro/templates/base.html

```diff
@@ -33,9 +33,8 @@
     {% if domain_verification_tag %}
       <meta name="google-site-verification" content="{{ domain_verification_tag }}" />
     {% endif %}
-    {% if zendesk_domain_verification_tag %}
-      <meta name='zd-site-verification' content="{{ zendesk_domain_verification_tag }}" />
-    {% endif %}
+    <meta name='zd-site-verification' content="o0itxdfh369m1bw4cuz9" />
+    <meta name='zd-site-verification' content="7e22kzikbruclk2jjomono" />
```

Comment:
I wasn't able to back track this value, could you refer me to the exact comment in the ticket?

Reply:
Just wanted to leave it here as well. It is mentioned in the email thread.